chia/util/streamable.py

```diff
@@ -339,6 +339,23 @@ def from_bytes(cls: Any, blob: bytes) -> Any:
         assert f.read() == b""
         return parsed
 
+    @classmethod
+    def fields_from_bytes(cls: Any, blob: bytes, fields_to_get: List) -> Any:
+        f = io.BytesIO(blob)
+        fields: Iterator[str] = iter(getattr(cls, "__annotations__", {}))
+        values: Iterator = (parse_f(f) for parse_f in PARSE_FUNCTIONS_FOR_STREAMABLE_CLASS[cls])
```

Comment:
is this really faster than just parsing the whole object? You still parse out all the values and store them in a tuple here. It seems like this would be about the same amount of work

Reply:
yes i think your right and i did it the wrong way

Reply:
after speaking with Slava it seems like we don't really have a way to only read an attribute without parsing it as we don't know the size and we don't have any separating operator between fields